docs/fastdds/library_overview/includes/exec_model.rst

```diff
@@ -26,66 +28,86 @@ Transport related threads (marked as UDP, TCP and SHM types) are only created wh
     * - Name
       - Type
       - Cardinality
+      - OS thread name
       - Description
     * - Event
       - General
       - One per DomainParticipant
+      - ``dds.ev.<participant_id>``
       - Processes periodic and triggered time events
     * - Discovery Server Event
       - General
       - One per DomainParticipant
+      - ``dds.ds_ev.<participant_id>``
       - Synchronizes access to the Discovery Server |br| Database
     * - Asynchronous Writer
       - General
-      - One per enabled asynchronous |br| flow controller
+      - One per enabled asynchronous |br| flow controller.
 
         Minimum 1.
+      - ``dds.asyn.<participant_id>.<async_flow_controller_index>``
```

Comment:
The table is cropped due to the new column. I think splitting this line (the longest in the column) might fix the problem